### PR TITLE
Added year of graduation field 

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
Made some improvements, implemented a drop-down

screen shot 2018-06-29 at 8 18 18 pm

screen shot 2018-06-29 at 8 18 52 pm

screen shot 2018-06-29 at 10 37 25 pm

Added a tooltip notifying to fill in the expected year of graduation for undergrads

### CODE REVIEW

File: migrations/055-to-056.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+alter table users add column "graduationYear" integer default 1990;
```
Comment (championswimmer) [inline migrations/055-to-056.sql]: default to NULL please

File: src/routers/signup.js [hunk 2]
```diff
@@ -28,6 +28,10 @@ router.post('/', makeGaEvent('submit', 'form', 'signup'), function (req, res) {
         req.flash('error', 'Password too weak. Use 5 characters at least.')
         return res.redirect('/signup')
     }
+    // if (req.body.graduationYear == null) {
```
Comment (championswimmer) [inline src/routers/signup.js]: remove commented out code please

Comment (VibhorCodecianGupta) [reply src/routers/signup.js]: Oh, my bad. Will do

**Comment by VibhorCodecianGupta** [comment]
@championswimmer done!

**Comment by VibhorCodecianGupta** [comment]
Alright, I'll do all that!

**Comment by championswimmer** [comment]
@VibhorCodecianGupta you can claim the bounty right now if you want to

**Comment by VibhorCodecianGupta** [comment]
@championswimmer refactored graduation year logic, please have a look :)

**Pull request closed without merging**